woocommerce/woocommerce-ios
# Shipment Tracking Mark 4: UI

Ref: #250 

This PR adds UI elements and logic to present Shipment Tracking in the `OrderDetailsViewController`.

At this moment, even though this PR includes the code to integrate with Yosemite to fetch tracking data, the data presented on screen is mocked until I figure out why that integration is not working (for me) as expected. ( p1551246523204900-slack-C6H8C3G23 )

![screen recording 2019-02-27 at 2 50 11 pm mov](https://user-images.githubusercontent.com/2722505/53471546-fff57c00-3a9f-11e9-8439-3fc26ea128ee.gif)

## Testing
As mentioned, at the moment the shipment data is mocked. Also, shipment data will be presented only for fulfilled orders, pending further discussion ( p1551231496200300-slack-C6H8C3G23 )

- Make sure you have a fulfilled order with one (or more) tracking numbers:
<img width="291" alt="screenshot 2019-02-27 at 3 02 27 pm" src="https://user-images.githubusercontent.com/2722505/53471789-c96c3100-3aa0-11e9-9e18-ab75ddcddbbe.png">

- Check out the branch
- Navigate to Orders > The order setup with tracking. Notice if mock tracking data is presented:

![simulator screen shot - iphone x - 2019-02-27 at 15 04 03](https://user-images.githubusercontent.com/2722505/53471848-f882a280-3aa0-11e9-874e-fa8a99dc76d6.png)

- Tap the "Track package" button for the HK Post entry. It should present a web view with http://automattic.com
- Tap the "Track package" button for the USPS Woo entry. It should present a web view with http://woocommerce.com

Mock data will be removed and testing instructions will be updated when this PR is ready for its final  review
